.github/workflows/win.yml

```
@@ -0,0 +1,18 @@
+name: Windows
+
+on: [push, pull_request]
+jobs:
+  windows:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: dependencies
+        run: |
+          choco install cmake
+      - name: build
+        run: |
+          cmake -H . -B build -G "Visual Studio 15 2017" -A x64
```

Sorry, you'll need to remove spaces here
```suggestion
          cmake -H. -Bbuild -G "Visual Studio 15 2017" -A x64
```